cilium-ubuntu-4.19.json

```diff
@@ -1,7 +1,7 @@
 {
   "variables": {
     "cpus": "8",
-    "disk_size": "40960",
+    "disk_size": "81920",
```

Comment:
For other reviewers: I've checked that this shouldn't impact the disk space we use on our Equinix servers.